src/templates/layouts/pressRelease/index.tsx

```diff
@@ -103,15 +108,57 @@ export const PressRelease = ({
                         if (!asset?.uri) {
                           return null
                         }
+                        let link = (
+                          <va-link
+                            data-testid="generic-file"
+                            filetype={asset.uri.split('.').pop()}
+                            href={asset.uri}
+                            download
+                            text={asset.name}
+                          />
+                        )
+                        switch (asset.type) {
```

Comment:
😅 There's _juuuuuust_ enough different for this to be frustrating. I this verbosity is best, but 🫠 .